### ISSUE TITLE
Floating number with leading zeros is wrongly treated as invalid

### ISSUE THREAD

**Issue by yuzie007** [microsoft/pyright#4226]
**Describe the bug**
The following should be a valid Python code, but pylance shows it as an invalid code.
```python
01.0
```
Floating numbers with leading zeros are sometimes useful for number alignment, as shown below, and thus I hope it should be recognized at least as a valid code.
```python
a = [
    01.0 / 16,
    09.0 / 16,
    07.0 / 16,
    15.0 / 16,
]
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by yuzie007** [comment]
Thank you so much for addressing the issue so promptly!

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.282, which I just published. It will also be included in a future release of pylance.
